R/data.R

```diff
@@ -53,3 +53,23 @@ NULL
 #' \item{year}{Year}
 #' \item{births}{Number of live births, rounded to nearest 1000}}
 "births"
+
+
+#' Return the head and tail of a data.freame
+#'
+#' For the purposes of DRY testing only.  Do not export.
+#'
+#' @param x
+#'
+#' @return data.frame
+#' @importFrom("utils", "head", "tail")
+first_last <- function(x) {
```

Comment:
This shouldn't be exported, and can just live in one of the test files.

I think you can eliminate the use of dplyr with something like:

```R
n <- nrow(x)
x[c(1:10, (n - 10):n)]
```

Reply:
Oh I was confused by the documentation